src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -137,22 +137,20 @@ const AdvancedSettings = ({ intl, courseId }) => {
               icon={Info}
               proctoringErrorsData={proctoringErrors}
               aria-hidden="true"
-              aria-labelledby={intl.formatMessage(messages.alertProctoringAriaLabelledby)}
-              aria-describedby={intl.formatMessage(messages.alertProctoringDescribedby)}
             />
           )}
           <TransitionReplace>
             {showSuccessAlert ? (
               <AlertMessage
-                key={intl.formatMessage(messages.alertSuccessAriaLabelledby)}
+                key="alert-confirmation-title"
                 show={showSuccessAlert}
                 variant="success"
                 icon={CheckCircle}
                 title={intl.formatMessage(messages.alertSuccess)}
                 description={intl.formatMessage(messages.alertSuccessDescriptions)}
                 aria-hidden="true"
-                aria-labelledby={intl.formatMessage(messages.alertSuccessAriaLabelledby)}
-                aria-describedby={intl.formatMessage(messages.alertSuccessAriaDescribedby)}
+                aria-label={intl.formatMessage(messages.alertSuccess)}
+                aria-description={intl.formatMessage(messages.alertSuccessDescriptions)}
```

Comment:
I think we still want to have `aria-labelledby` here, but just use the element id directly instead of using `intl.formatMessage` for it

Reply:
Yes it's one option, the thing is that two ids would need to pass from parent down to Alert componenet, I have an example here https://github.com/openedx/frontend-app-course-authoring/pull/1162/commits/f56f44b443e73498e8422e09aa3047ab1fe62a25 , if you think it's okay then I guess we do the same for all other usage?